exporter/doc/measure_section.go

```diff
@@ -0,0 +1,121 @@
+package doc
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+
+	"github.com/projectcypress/cdatools/models"
+)
+
+type MeasureSection struct {
+	models.Measure
+	Results         map[string]AggregateCount
+	PerformanceRate PopulationGroup
+	Data            MeasureData
+}
+
+type AggregateCount struct {
+	Measure_HQMFID   string
+	Populations      []Population
+	PopulationGroups []PopulationGroup
+}
+
+func (ac AggregateCount) IsCV() bool {
```

Comment:
Ah ok. I don't think we need to adhere strictly to HDS function names though. Still would like to see this spelled out